<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCapture</class>
 <widget class="QMainWindow" name="CameraCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3">
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>145</red>
                 <green>145</green>
                 <blue>145</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>145</red>
                 <green>145</green>
                 <blue>145</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>145</red>
                 <green>145</green>
                 <blue>145</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>145</red>
                 <green>145</green>
                 <blue>145</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lastImagePreviewLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <item>
          <widget class="QToolButton" name="startCameraButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Start camera</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="takeImageButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Take Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
         <item>
          <widget class="QToolButton" name="stopButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pauseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="recordButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="text">
            <string>Record</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="actionAudio"/>
   </widget>
   <widget class="QMenu" name="menuFlash">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuFlash_On">
     <property name="title">
      <string>Flash</string>
     </property>
     <addaction name="actionFlash_On_2"/>
     <addaction name="actionFlash_Off"/>
     <addaction name="actionFlash_auto"/>
     <addaction name="actionRed_eye"/>
     <addaction name="actionFlash_FillIn"/>
    </widget>
    <widget class="QMenu" name="menuFocus">
     <property name="title">
      <string>Focus</string>
     </property>
     <addaction name="action_Focus_On"/>
     <addaction name="action_Focus_Off"/>
    </widget>
    <widget class="QMenu" name="menuExposure">
     <property name="title">
      <string>Exposure</string>
     </property>
     <addaction name="actionNight"/>
     <addaction name="actionBacklight"/>
     <addaction name="actionSport"/>
     <addaction name="actionBeach"/>
    </widget>
    <widget class="QMenu" name="menuWhiteBalance">
     <property name="title">
      <string>WhiteBalance</string>
     </property>
     <addaction name="actionAuto"/>
     <addaction name="actionSunlight"/>
     <addaction name="actionCloudy"/>
     <addaction name="actionTungsten"/>
    </widget>
    <addaction name="menuFlash_On"/>
    <addaction name="menuFocus"/>
    <addaction name="menuExposure"/>
    <addaction name="menuWhiteBalance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevices"/>
   <addaction name="menuFlash"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionFlash_On_2">
   <property name="text">
    <string>Flash On</string>
   </property>
  </action>
  <action name="actionFlash_Off">
   <property name="text">
    <string>Flash Off</string>
   </property>
  </action>
  <action name="actionFlash_auto">
   <property name="text">
    <string>Flash Auto</string>
   </property>
  </action>
  <action name="actionRed_eye">
   <property name="text">
    <string>Red Eye</string>
   </property>
  </action>
  <action name="actionFlash_FillIn">
   <property name="text">
    <string>Flash FillIn</string>
   </property>
  </action>
  <action name="action_Focus_On">
   <property name="text">
    <string>Focus On (auto)</string>
   </property>
  </action>
  <action name="action_Focus_Off">
   <property name="text">
    <string>Focus Off (auto)</string>
   </property>
  </action>
  <action name="actionNight">
   <property name="text">
    <string>Night</string>
   </property>
  </action>
  <action name="actionBacklight">
   <property name="text">
    <string>Backlight</string>
   </property>
  </action>
  <action name="actionSport">
   <property name="text">
    <string>Sport</string>
   </property>
  </action>
  <action name="actionBeach">
   <property name="text">
    <string>Beach</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionSunlight">
   <property name="text">
    <string>Sunlight</string>
   </property>
  </action>
  <action name="actionCloudy">
   <property name="text">
    <string>Cloudy</string>
   </property>
  </action>
  <action name="actionTungsten">
   <property name="text">
    <string>Tungsten</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CameraCapture</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>CameraCapture</receiver>
   <slot>settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startCameraButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCapture</receiver>
   <slot>toggleCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCapture</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCapture</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCapture</receiver>
   <slot>record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>takeImageButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraCapture</receiver>
   <slot>takeImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>record()</slot>
  <slot>pause()</slot>
  <slot>stop()</slot>
  <slot>enablePreview(bool)</slot>
  <slot>settings()</slot>
  <slot>takeImage()</slot>
  <slot>toggleCamera()</slot>
 </slots>
</ui>
